packages/database/src/models/Invoice/Invoice.ts

```diff
@@ -186,4 +191,49 @@ export class Invoice extends Model {
       },
     });
   }
+
+  static async initializeInvoice(
+    encounter: Encounter,
+    facilitySettings: ReadSettings,
+    userId: string,
+    data: any,
+  ) {
+    const insurerId = await this.sequelize.models.PatientAdditionalData.findOne({
+      where: { patientId: encounter.patientId },
+      attributes: ['insurerId'],
+    }).then(patientData => patientData?.insurerId);
+    const insurerPercentage = await facilitySettings.get(
+      SETTING_KEYS.INSURER_DEFAULT_CONTRIBUTION as SettingPath,
+    );
+    const defaultInsurer =
```

Comment:
As we discussed on the other PR, patient insurers aren't supported at the moment so we can remove the code relating to them which will simplify this a bit.